.github/workflows/release-notes.yaml

```
@@ -0,0 +1,75 @@
+  release-notes:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: ⤵️ Check out code from GitHub
+        uses: actions/checkout@v4.2.1
+
+      - name: Get information
+        id: info
+        uses: home-assistant/actions/helpers/info@master
+        with:
+          path: ./${{ matrix.addon }}
```

actionlint

property "addon" is not defined in object type {}

(expression)